node_modules/oae-messagebox/lib/api.js

```diff
@@ -63,7 +63,16 @@ var _replaceLinks = function(body) {
     var regex = new RegExp('(.?)https?://(' + hosts.join('|') + ')(/[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])(.?)', 'gi');
 
     // Replace any matched URLs with relative links in markdown format
-    return body.replace(regex, function(fullMatch, preURLChar, host, path, postURLChar) {
+    return body.replace(regex, function(fullMatch, preURLChar, host, path, postURLChar, offset) {
+        // If there are an odd number of backtics before the match it's inside a quote and should be left as is
+        var inQuote = body.substring(0, offset + 1).split('`').length % 2 === 0;
+        // If the line the match is on starts with 4 spaces it's a block quote and should be left as is
+        var inBlockQuote = body.substr(body.substring(0, offset + 1).lastIndexOf('\n') + 1, 4) === '    ';
```

Comment:
Ah, looks like it needs a blank line before the indented lines. That's going to make the fix a little more complex.